### PR TITLE
6 getcommentbyarticleid

### PR DESCRIPTION
**Pull request by Hartooro**

### CODE REVIEW

**Comment by Hartooro** [comment]
removed console.log on line 118 of app.test.js

File: __tests__/app.test.js [hunk 1]
```diff
@@ -85,4 +86,37 @@ describe('4 get/api/articles/:article_id', () => {
...
+  describe('5 get/api/articles/id/comments', () => {
+    test('returns 200, with the comments related to the article id', () => {
+        return request(app).get("/api/articles/5/comments").expect(200)
+        .then((res)=>{
+            expect(res.body.comments.length).toBe(2)
+            res.body.comments.forEach((comment)=>{
+            
+               expect(comment).toEqual(expect.objectContaining({
+                comment_id: expect.any(Number),
+                article_id : 5,
```
Comment (Slyose) [inline __tests__/app.test.js]: The comment doesn't need to return the article id

File: __tests__/app.test.js [hunk 2]
```diff
@@ -85,4 +86,37 @@ describe('4 get/api/articles/:article_id', () => {
...
+            })
+        })
+    })
+    test('tests for a valid article, but no comments are found, so should return an empty array', () => {
+        return request(app).get("/api/articles/4/comments").expect(200).then((res)=>{
+            expect(res.body.comments.length).toBe(0)
+            expect(res.body.comments).toEqual([])
+        })
+    });
+  });
```
Comment (Slyose) [inline __tests__/app.test.js]: Needs a couple extra tests for: 
- an invalid id (e.g /api/articles/banana/comments)
- an id of a non - existant endpoint

File: query.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
-------------+-------------------------------------+-------+---------------+-----------------------------------+---------------------+-------
-          1 | Living in the shadow of a great man | mitch | butter_bridge | I find this existence challenging | 2020-07-09 21:11:00 |   100
+ article_id |        title        | topic |  author  |                                                                                     body                                                                                      |     created_at      | votes 
+------------+---------------------+-------+----------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------+-------
+          4 | Student SUES Mitch! | mitch | rogersop | We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages | 2020-05-06 02:14:00 |     0
 (1 row)
```
Comment (Slyose) [inline query.txt:6]: This can also be .gitignored

File: query.sql [hunk 4]
```diff
@@ -1,5 +1,5 @@
 \c nc_news_test
 
 Select * from articles
-where article_id = 1;
+where article_id = 4;
```
Comment (Slyose) [inline query.sql:5]: This whole file should be .gitignored, as it's only a practice file it shouldn't reach the main branch, or even git hub at all.

**Review by Slyose — CHANGES_REQUESTED** [review]

File: __tests__/app.test.js (hunk 1 above)
Comment (Hartooro) [reply __tests__/app.test.js]: Isn't it a good way to check whether it's pulling the right comments? @Slyose

Comment (Slyose) [reply __tests__/app.test.js:101]: image

Comment (Slyose) [reply __tests__/app.test.js:101]: @Hartooro

**Pull request merged**